recipe/meta.yaml

```diff
@@ -47,12 +47,118 @@ test:
   requires:
     - lhapdf
     - hepmc2
-  files:
-    - $PREFIX/bin/Sherpa
-    - $PREFIX/bin/Sherpa-config
   source_files:
     - Examples
   commands:
+    - test -f $PREFIX/bin/Sherpa
+    - test -f $PREFIX/bin/sconsloops  # FIXME: Broken (from MadLoop addon)
+    - test -f $PREFIX/bin/get_bibtex
+    - test -f $PREFIX/bin/Sherpa-config
+    - test -f $PREFIX/bin/init_nlo.sh
+    - test -f $PREFIX/bin/plot_graphs.sh
+    - test -f $PREFIX/bin/make2scons  # FIXME: Has conda-forge build paths baked in
+    - test -f $PREFIX/bin/plot_stats.sh
+
+    - test -f $PREFIX/lib/SHERPA-MC/libPhasicProcess.so.0  # [linux]
```

Comment:
As we don't know the full file names on macOS yet, I can't yet do something like

```yaml
$PREFIX/lib/SHERPA-MC/libPhasicProcess${SHLIB_EXT}
```

though I also don't know how to get around the `.0` at the end, as I don't think(?) macOS will have that.